### PR TITLE
Clean up database provider list of MC

### PR DESCRIPTION
**Pull request by unkcpz**
According to Nicola, he wants to have MC3D and MC2D on the main list.
I also change the name of subgroup 'General' -> 'Main Projects' and 'Projects' -> 'Contribution Projects'.
Any idea what is the more proper name for the list? pinning @giovannipizzi, @csadorf @eimrek @CasperWA for comment about the name.

### CODE REVIEW

**Comment by unkcpz** [comment]
This change also needs to make for QeApp which is inconvenient as described by @csadorf  https://github.com/aiidalab/aiidalab-qe/issues/155#issuecomment-972934387

I think the question is in which repository we put the static file (txt, or JSON) as the default group list. In my opinion, in order that the list to be used by 'downstream' package, it need to be put in `voila-optimade-client` and used by `tools-optimade-client` here and by `aiidalab-base-widget` for QeApp.
In this case, may be it is a quick implementation to just create a dict `DEFAULT_DATABASE_GROUPING` as const variable in `optimade_client` library contained and installed by `voila-optimade-client`.

What do you think? @CasperWA

File: OPTIMADE-Client.ipynb [hunk 1]
```diff
@@ -121,11 +119,25 @@
...
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
```
Comment (CasperWA) [inline OPTIMADE-Client.ipynb:135]:
```suggestion
```

File: OPTIMADE-Client.ipynb [hunk 2]
```diff
@@ -121,11 +119,25 @@
...
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3 (ipykernel)",
```
Comment (CasperWA) [inline OPTIMADE-Client.ipynb:140]:
```suggestion
   "display_name": "Python 3",
```

**Review by CasperWA — CHANGES_REQUESTED** [review]
Seems fine to me.
I'm just removing two empty code cells and reverting an unnecessary change in the kernel's display name.

**Comment by eimrek** [comment]
Hi @unkcpz, I think the names you proposed work well. But i would use "Contributed projects" instead of "Contribution projects".

Also, just to be aware, the names of the two main optimade databases will be changed to "mc3d" and "mc2d", after the aiida-2.0 migration: https://github.com/materialscloud-org/openstack-ansible/pull/154

File: OPTIMADE-Client.ipynb (hunk 2 above)
Comment (unkcpz) [reply OPTIMADE-Client.ipynb]: changed.

**Comment by unkcpz** [comment]
@CasperWA @eimrek 

thanks for the review! Updated upon your suggestion. 

>  just to be aware, the names of the two main optimade databases will be changed to "mc3d" and "mc2d", after the aiida-2.0 migration

Yes, I will keep it in mind after that we need also to rerun `mc_optimade_index` role.

**Comment by eimrek** [comment]
Looks good! But could you push this to dev-dokku to demonstrate the changes and that it works as intended? (as i understand, that's what the dev machine is for, right?)

**Comment by unkcpz** [comment]
@eimrek Thanks! Once merged, it will automatically be deployed to the dev server by the CI action. https://github.com/materialscloud-org/tools-optimade-client/blob/master/.github/workflows/cd_dev-dokku.yml

**Comment by unkcpz** [comment]
Seems works fine https://optimadeclient.matcloud.xyz/

**Pull request merged**